core/src/main/resources/jenkins/monitor/JavaVersionRecommendationAdminMonitor/message.jelly

```diff
@@ -25,7 +25,8 @@ THE SOFTWARE.
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:l="/lib/layout" xmlns:f="/lib/form">
     <div class="alert alert-info">
-        ${%Recommended_Java_Version}
+        <j:invokeStatic var="currentRuntimeJavaVersion" className="jenkins.util.java.JavaUtils" method="getCurrentRuntimeJavaVersion" />
```

Comment:
slight over engineering I think, the monitor only works with java 8 but I guess no harm and could be useful later

Reply:
Good communication about migrations ought to include at least two pieces of information, I think: what you're currently using, and what you should be using. Yes the monitor only works with Java 8, but including the Java version in the message ought to help users understand more clearly what needs to change on their end.

Reply:
I get that, hardcoding java 8 would work too though 😄. no issue with this though just was a bit confused it was dynamically getting something that is always going to be 8 (with the current code)